### DISCUSSION TITLE
Is it supported for Android 15?

### DISCUSSION THREAD

**Discussion by llv22** [bmax121/APatch#719] [Q&A]
Could you help clarify?

**Comment by marat2509** [comment]
In theory, the mechanism of APatch operation depends not on the Android version, but on the version and architecture of the kernel.

**Reply by llv22** [reply]
That's why I guess maybe not support, as Android 15 kernel version is 6.6

**Reply by marat2509** [reply]
At the expense of the kernel version 6.6 - is the version of the Generic Kernel Image of the kernel. If I'm not mistaken, APatch does not support working with GKI kernels at all (after all, KernelSU is designed for them)

**Reply by llv22** [reply]
@marat2509 so in Android 15, APatch will help no longer?

**Reply by bmax121** [reply]
try lastes CI version, KernelPatch is supported 6.6 now

**Reply by llv22** [reply]
@bmax121 could you share the link to me?
